dev/domain-controller/new-agent-vm/Install-PCoIPAgent.sh

```diff
@@ -0,0 +1,210 @@
+#!/bin/bash
+
```

Comment:
We should be using `set -eu` in shell scripts to ensure that the script exits on a) uncaught errors and b) if variables are undefined. (For reasons why this is a good idea, google for Valve blowing away user's home directories because of an unset variable in an installer script ;) )